oap-server/server-storage-plugin/storage-iotdb-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/iotdb/IoTDBClient.java

```diff
@@ -0,0 +1,342 @@
+import org.apache.skywalking.oap.server.core.storage.StorageHashMapBuilder;
+import org.apache.skywalking.oap.server.library.client.Client;
+import org.apache.skywalking.oap.server.library.client.healthcheck.DelegatedHealthChecker;
+import org.apache.skywalking.oap.server.library.client.healthcheck.HealthCheckable;
+import org.apache.skywalking.oap.server.library.util.HealthChecker;
+import org.apache.skywalking.oap.server.storage.plugin.iotdb.base.IoTDBInsertRequest;
+
+@Slf4j
+public class IoTDBClient implements Client, HealthCheckable {
+    private final DelegatedHealthChecker healthChecker = new DelegatedHealthChecker();
+    private final IoTDBStorageConfig config;
+
+    private SessionPool sessionPool;
+    private final String storageGroup;
+
+    public static final String DOT = ".";
+    public static final String ALIGN_BY_DEVICE = " align by device";
+
+    public static final String TIME_BUCKET = "time_bucket";
+    public static final String TIME = "Time";
+    public static final String TIMESTAMP = "timestamp";
+
+    public static final String ID_IDX = "id";
+    public static final String ENTITY_ID_IDX = "entity_id";
+    public static final String NODE_TYPE_IDX = "node_type";
+    public static final String SERVICE_ID_IDX = "service_id";
+    public static final String GROUP_IDX = "service_group";
+    public static final String TRACE_ID_IDX = "trace_id";
+
+    public IoTDBClient(IoTDBStorageConfig config) throws IOException {
+        this.config = config;
+        storageGroup = config.getStorageGroup();
+    }
+
+    @Override
+    public void connect() throws IoTDBConnectionException, StatementExecutionException {
+        try {
+            sessionPool = new SessionPool(config.getHost(), config.getRpcPort(), config.getUsername(),
+                    config.getPassword(), config.getSessionPoolSize());
+            sessionPool.setStorageGroup(storageGroup);
+
+            healthChecker.health();
+        } catch (StatementExecutionException e) {
+            if (e.getStatusCode() != TSStatusCode.PATH_ALREADY_EXIST_ERROR.getStatusCode()) {
+                healthChecker.unHealth(e);
+                throw e;
+            }
+        }
+    }
+
+    @Override
+    public void shutdown() {
+        sessionPool.close();
+        this.healthChecker.health();
+    }
+
+    @Override
+    public void registerChecker(HealthChecker healthChecker) {
+        this.healthChecker.register(healthChecker);
+    }
+
+    public SessionPool getSessionPool() {
+        return sessionPool;
+    }
+
+    public IoTDBStorageConfig getConfig() {
+        return config;
+    }
+
+    /**
+     * Write data to IoTDB
+     *
+     * @param request an IoTDBInsertRequest
+     * @throws IOException IoTDBConnectionException or StatementExecutionException
+     */
+    public void write(IoTDBInsertRequest request) throws IOException {
+        if (log.isDebugEnabled()) {
+            log.debug(request.toString());
```

Comment:
Can you add some descriptive message to the log so that we can search them when needed, such as

```suggestion
            log.debug("Writing data to IoTDB: {}", request);
```